packages/web-app-files/src/views/trash/Overview.vue

```diff
@@ -175,7 +174,27 @@ export default defineComponent({
     ])
 
     const getSpaceName = (space: SpaceResource) => {
-      return isPersonalSpaceResource(space) ? $gettext('Personal') : space.name
+      if (isPersonalSpaceResource(space)) {
+        return $gettext('Personal')
+      }
+      if (space.disabled === true) {
+        return $gettext('%{spaceName} (disabled)', { spaceName: space.name })
+      }
+
+      return space.name
+    }
+
+    const getSpaceAttributes = (space: SpaceResource) => {
+      if (isProjectSpaceResource(space) && space.disabled === true) {
+        return {
+          disabled: true
```

Comment:
Is `type: 'span'` needed?

```suggestion
          type: 'span',
          disabled: true
```

Reply:
Could you check that in the ui, I don't see any reason why it should be

Reply:
Hehe, committed w/o checking and nope the ui representation is quite wrong with a span, because it doesn't represent the disabled state, reverting

Reply:
I WAS TESTING YOU :trollface:

Reply:
TRY HARD 🔥🔥🔥